dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/BaseContainersKit.scala

```diff
@@ -78,6 +78,17 @@ trait BaseContainersKit extends ScorexLogging {
 
   protected implicit val tryHttpBackend: LoggingSttpBackend[Try, Nothing] = new LoggingSttpBackend[Try, Nothing](
     TryHttpURLConnectionBackend(customizeConnection = conn => {
+      // For tests with a high latency
+      conn.setConnectTimeout(30000)
+      conn.setReadTimeout(30000)
+
+      conn.setDefaultUseCaches(false)
+      conn.setUseCaches(false)
+      conn.setRequestProperty("Cache-Control", "no-store")
+      conn.setRequestProperty("Pragma", "no-cache")
+      conn.setRequestProperty("If-Modified-Since", "Sat, 1 Jan 2000 00:00:00 GMT")
```

Comment:
wtf?

Reply:
Fighting caches :) There could be any date in the past, doesn't matter: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since , syntax:
> If-Modified-Since: <day-name>, <day> <month> <year> <hour>:<minute>:<second> GMT

I tried to specify all parameters to avoid HTTP caches by the Java's client. But it seems this doesn't help tests on my Mac. But it works well on CI. Wondering 🌈